example/lib/meeting/meeting_store.dart

```diff
@@ -1295,13 +1327,53 @@ class MeetingStore extends ChangeNotifier
   }
 
   void enterPipMode() async {
-    //to check whether pip is available
-    bool _isPipAvailable = await _hmsSDKInteractor.isPipAvailable();
-    if (_isPipAvailable) {
-      //[isPipActive] method can also be used to check whether application is in pip Mode or not
-      isPipActive =
-          await _hmsSDKInteractor.enterPipMode(autoEnterPip: isPipAutoEnabled);
-      notifyListeners();
+    //to check whether pip is available in android
+    if (Platform.isAndroid) {
+      bool _isPipAvailable = await HMSAndroidPIPController.isAvailable();
+      if (_isPipAvailable) {
+        //[isPipActive] method can also be used to check whether application is in pip Mode or not
+        isPipActive =
+            await HMSAndroidPIPController.start(autoEnterPip: isPipAutoEnabled);
+        notifyListeners();
+      }
+    }
+  }
+
+  Future<bool> isPIPActive() async {
```

Comment:
rename function to avoid Android / iOS confusion for readers
```suggestion
  Future<bool> isPIPActiveOnAndroid() async {
```